public/app/percona/shared/components/Elements/TabbedContent/TabbedContent.types.ts

```diff
@@ -0,0 +1,22 @@
+export interface ContentTab {
+  label: string;
+  key: string;
+  disabled?: boolean;
+  component: () => JSX.Element;
```

Comment:
Are there any advantages of using `JSX.Element` instead of `ReactNode`?

Reply:
To be honest I didn't even know the exact difference between them.
After looking at this: https://stackoverflow.com/questions/58123398/when-to-use-jsx-element-vs-reactnode-vs-reactelement

It seems functional components return a `ReactElement`, while a `ReactNode` can be a `ReactElement` but also pretty much everything else such as null, undefined or boolean.
A `JSX.Element` is just a `ReactElement` with `any` types.
Since we always use FC, seems like maybe `JSX.Element` is the correct choice, right?

Reply:
Based on that description I would use `ReactNode` in most cases, because it has `props` and `type` and it can be `null` or any primitive. For tab components `JSX.Element` seems ok. I'm not seeing a case where the tab can be `null` or anything else rather than a component.